pom.xml

```diff
@@ -516,28 +510,6 @@
                 <artifactId>httpcore</artifactId>
                 <version>${version.apache.http}</version>
             </dependency>
-
```

Comment:
why did you remove these? they're dependency management so, they won't be brought it unless asked for. By having them here, we have a single point of version management, no?
Speaking of version managements, I think we should move all dependencies into the `dependencyManagement` section and have individual modules bring in only what they need